GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -527,6 +529,7 @@ private void InitializeComponent()
             this.fetchPruneAllToolStripMenuItem.Name = "fetchPruneAllToolStripMenuItem";
             this.fetchPruneAllToolStripMenuItem.Size = new System.Drawing.Size(239, 22);
             this.fetchPruneAllToolStripMenuItem.Text = "Fetch and prune all";
+            this.fetchPruneAllToolStripMenuItem.ToolTipText = "Fetch branches and tags from all remote repositories also prune deleted refs";
```

Comment:
Might be nice to add a bit more detail here for people who don't know what pruning is.

> Fetch branches and tags from all remote repositories, then remove (prune) any remote-tracking references that no longer exist on remotes.

Does this command also run `--prune-tags`?

Reply:
\<sarcasm>Gosh, to get an answer to the `--prune-tags` question I had to press F12 so many times that the label on F12 key almost faded away.\</sarcasm>
And the answer is no. So how about this:
>Fetch branches and tags from all remote repositories, then remove (prune) any remote-tracking branch references that no longer exist on remotes.

Reply:
Actually.. `--prune-tags` is not found anywhere in solution. But if we decide to add it, lets to it in another PR.

Reply:
Thanks for checking. My phone doesn't have an F12 key.